tests/lib/ignored-paths.js

```diff
@@ -244,6 +244,12 @@ describe("IgnoredPaths", () => {
             assert.strictEqual(ignoredPaths.ignoreFileDir, path.dirname(ignoreFilePath));
         });
 
+        it("should set the common ancestor directory of cwd and ignorePath", () => {
+            const ignoredPaths = new IgnoredPaths({ ignore: true, ignorePath: ignoreFilePath, cwd: path.resolve(ignoreFilePath, "../../../testcwd") });
+
+            assert.strictEqual(ignoredPaths.getBaseDir(), path.resolve(ignoreFilePath, "../../.."));
+        });
+
```

Comment:
Can you add tests for an ignore file with `\r\n`? (Git would convert the linebreaks to `\n` when the repository is cloned, so it would probably be necessary to create the file at runtime.)

Reply:
I added it.